src/ofxstatement/statement.py

```diff
@@ -166,6 +189,49 @@ def generate_transaction_id(stmt_line):
     return h.hexdigest()
 
 
+def generate_unique_transaction_id(stmt_line, unique_id_set: set):  # pragma: no cover
```

Comment:
Argh, I missed this :) This is not how you increase test code coverage, but ok, next time :)